sources/packages/web/src/views/institution/locations/active-applications/ActiveApplicationSummaryData.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <body-header
+    title="Applications"
+    :recordsCount="applications.results?.length"
+    class="m-1"
+  >
+    <template #actions>
+      <div class="search-box-display-width">
+        <v-text-field
+          label="Search name or application #"
+          prepend-inner-icon="mdi-magnify"
+          variant="outlined"
+          v-model="searchCriteria"
+          @keyup.enter="searchActiveApplications"
+        ></v-text-field>
+      </div>
+    </template>
+  </body-header>
+  <br />
+  <content-group>
+    <DataTable
+      :value="applications.results"
+      :lazy="true"
+      class="p-m-4"
+      :paginator="true"
+      :rows="pageLimit"
+      :rowsPerPageOptions="rowsPerPageOptions"
+      :totalRecords="applications.count"
+      @page="pageEvent"
+      @sort="sortEvent"
+    >
+      <template #empty>
```

Comment:
Please use the `toggle-content`. Please check the below reference:
sources\packages\web\src\components\aest\students\assessment\History.vue